client/me/purchases/remove-purchase/index.jsx

```diff
@@ -29,9 +45,35 @@ const RemovePurchase = React.createClass( {
 	},
 
 	getInitialState() {
+		// shuffle reason order, but keep another_reason_one last
+		const questionOneOrder = shuffle( [
+			'could_not_install',
+			'too_hard',
+			'did_not_include',
+			'only_need_free'
+		] );
+		questionOneOrder.push( 'another_reason_one' );
+
+		const questionTwoOrder = shuffle( [
+			'staying_here',
+			'other_wordpress',
+			'different_service',
+			'no_need'
+		] );
+		questionTwoOrder.push( 'another_reason_two' );
+
 		return {
 			isDialogVisible: false,
-			isRemoving: false
+			isSurveyOneVisible: config.isEnabled( 'upgrades/removal-survey' ),
+			isSurveyTwoVisible: ! config.isEnabled( 'upgrades/removal-survey' ),
```

Comment:
I'm confused by what is happening here. We're setting these values as determined by a config setting, but we have methods below that override the state values.

What do these two surveys represent and what is this config setting supposed to determine? Normally we use config settings like this to completely hide a new feature for a given environment when the config setting is missing or false. But it appears that we are adding a new dialog whether this value is true or false.